### DISCUSSION TITLE
Concern about Bearer Token Exposure when Using Directus Integration Module with Nuxt 3

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [Intevel/nuxt-directus#204] [Q&A]
Hello,

I'm using your integration module for Directus with Nuxt 3, and I've placed the token within the **runtimeConfig.directus.token**. However, I've noticed that the token is being exposed when the Nuxt site loads. Please note that I'm using **ssr:true**.

Is this kind of bearer token exposure normal? Could you recommend any security techniques to prevent my token from being exposed to the internet?

Thank you.
![Captura de tela de 2023-08-30 15-34-53](https://github.com/Intevel/nuxt-directus/assets/379850/902cb30e-9f1e-4d24-b739-58bc8ccaf96d)

**Comment by Intevel** [comment]
So the problem is, that the token has to be sent within the request, if a request ist performed on server side, this should be fine.

**Comment by hermesalvesbr** [comment]
How can I do this? SSR it's not for this?
